controller/utils/utils.go

```diff
@@ -39,6 +39,7 @@ func JobConfig(f *ct.ExpandedFormation, name, hostID string, uuid string) *host.
 	metadata["flynn-controller.app"] = f.App.ID
 	metadata["flynn-controller.app_name"] = f.App.Name
 	metadata["flynn-controller.release"] = f.Release.ID
+	metadata["flynn-controller.formation"] = f.App.ID + ":" + f.Release.ID
```

Comment:
I'm not sure I follow this. This appears to be redundant information, since the AppID and ReleaseID are already in the meta. Do we never use `JobConfig` for one-off jobs? Can we mark the jobs more explicitly instead of repeating existing information?